### PR TITLE
[PowerToys]Add GPO rules for Advanced Paste

### PR DESCRIPTION
**Pull request by jaimecbernardo**
Adds the missing GPO policies for Advanced Paste.

One of them is already out, but let's release this PR only after 0.81.1 releases.

@htcfreek , can you please review if you have time? I'm not sure about Intune information, but I tried to use what I saw from PowerToys Run.

### CODE REVIEW

File: hub/powertoys/grouppolicy.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ These policies have a higher priority than, and will override, the policy "Confi
 
 | Utility | ADMX GP name |ADMX GP unique name /</br>Registry value name /</br>Intune PolicyID |
 | :--- | :--- | :--- |
+|Advanced Paste|Advanced Paste: Configure enabled state|ConfigureEnabledUtilityAdvancedPaste|
```
Comment (htcfreek) [inline hub/powertoys/grouppolicy.md:96]: Please remove the old one from "Paste as Plaintext". Or should we add a `supported from ... to ... note`?

**Review by htcfreek — CHANGES_REQUESTED** [review]
See comment.

For the policy details looking good. (Didn't checked against the admx for typos.)

File: hub/powertoys/grouppolicy.md (hunk 1 above)
Comment (jaimecbernardo) [reply hub/powertoys/grouppolicy.md:96]: Good spotting! Just pushed. Thank you!

**Review by alvinashcraft — APPROVED** [review]
This looks good. Let me know when this patch is released, and I'll merge the PR.

**Comment by jaimecbernardo** [comment]
@alvinashcraft Thank you for the review. 0.81.1 is released. This can go in 😉  Thank you!

**Pull request merged**